server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DBTableController.java

```diff
@@ -122,4 +123,15 @@ public ListResponse<DataType> getPartitionKeyDataTypes(@PathVariable String sess
         return Responses.list(this.partitionPlanService.getPartitionKeyDataTypes(sessionId, databaseId, tableName));
     }
 
+    @PutMapping(value = "/{sessionId}/databases/{databaseName}/externalTables/"
```

Comment:
Shoud it be `@PostMapping`?

Reply:
`@PutMapping` stands for update, `@PostMapping` stands for new, I think `@PutMapping` is more appropriate

Reply:
use @PostMapping